extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetTableLocationKey.java

```diff
@@ -70,7 +73,7 @@ public ParquetTableLocationKey(@NotNull final URI parquetFileUri, final int orde
     }
 
     private static URI validateParquetFile(@NotNull final File file) {
-        return validateParquetFile(file.toURI());
+        return validateParquetFile(convertFileToURI(file, false));
```

Comment:
Have you done a search for `toURI` in our code to catch any stray usages?

Reply:
There's a use in QueryCompiler; I think it's less critical b/c we are already doing FS ops in the same code path:

```java
        if (!file.mkdirs() && !file.isDirectory()) {
            throw new UncheckedDeephavenException(runtimeErrMsg.get());
        }
```

If we feel strongly, we can loop in Nate.

Reply:
And for the Path#toUri, we make this call in `JsPluginsZipFilesystem create()` but again, we are creating some directories in the same method, so I skipped it. Let me know if you want to change this.

Reply:
These are both "low frequency" usages. No need to change them at this time.